src/verifier.js

```diff
@@ -78,7 +78,7 @@ class LocalVerifier {
         const payload = { [`${this.options.entity}Id`]: id };
         done(null, entity, payload);
       })
-      .catch(error => error ? done(error) : done(null, error));
+      .catch(error => error ? done(error) : done(null, error, { message: 'Invalid login' }));
```

Comment:
I think for some passport strategies they respond differently based on the `fail` method being called vs. the `error` method. https://github.com/jaredhanson/passport-local/blob/master/lib/strategy.js#L81

Just trying to track it down to see if there is a difference.